include/picongpu/param/speciesInitialization.param

```diff
@@ -20,12 +20,10 @@
 /** @file
  *
  * Initialize particles inside particle species. This is the final step in
- * setting up particles (defined in speciesDefinition.param) via density profiles
- * (defined in density.param). One can then further derive particles from one
- * species to an other and manipulate attributes with "manipulators" and "filters"
- * (defined in particle.param and particleFilters.param).
- *
- * For a full list of options, see the user manual section "Usage" - "Particles".
+ * setting up particles (defined in `speciesDefinition.param`) via density
+ * profiles (defined in `density.param`). One can then further derive particles
+ * from one species to an other and manipulate attributes with "manipulators"
```

Comment:
*another*

Reply:
Sorry, should have just mentioned this once for all `speciesInitialization.param` :sweat_smile:

Reply:
fixed in all places :)